src/components/reusable/card/card.ts

```diff
@@ -0,0 +1,145 @@
+import { LitElement, html } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+
+import CardScss from './card.scss';
+
+/**
+ * Card.
+ * @slot card-logo - Slot for card logo (Required). Size 32 * 32 as per UX guidelines.
+ * @slot card-action-button - Slot for Card action button.
+ * @slot card-overflow-menu - Slot for overflow menu.
+ * @slot card-thumbnail - Slot for card thumbnail image.
+ * @slot tags - Slot for the tag component.
+ * @slot card-links - Slot for card links.
+ * @slot unnamed - Slot for card body content.
+ */
+
+@customElement('kyn-card')
+export class Card extends LitElement {
+  static override styles = CardScss;
+  /** Card width. Default 264px. */
+  @property({ type: Number })
+  cardWidth = 264;
+
+  @property({ type: String })
+  cardLink = '/';
+
+  /** Card Type. `'normal'` & `'clickable'` */
+  @property({ type: String })
+  type = 'normal';
+
+  @property({ type: String })
+  cardTitle = '';
+
+  /** Card Subtitle. */
+  @property({ type: String })
+  subTitle = '';
+
+  /** Card Description. */
+  @property({ type: String })
+  description = '';
+
+  /** Card's thumbnail image position. `top` & `middle`. */
+  @property({ type: String })
+  imagePosition = 'top';
+
+  /** Show action button */
+  @property({ type: Boolean })
+  showActionBtn = false;
+
+  /** Show Option Menu. use only with non-clickable cards */
+  showOptionMenu = false;
+
+  /** Card collapse state.
+   * @ignore
+   */
+  @state()
+  isCollapse = false;
+
+  override render() {
+    return html`${this.type === 'clickable'
+      ? html`<a
+          href=${this.cardLink}
+          target="_blank"
```

Comment:
We shouldn't force new tab here. We also will need to add an event to capture link clicks.